sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeReceiptsOperation.cs

```diff
@@ -55,13 +86,19 @@ internal ExtractReceiptOperation(ServiceClient operations, string operationLocat
         public override async ValueTask<Response> UpdateStatusAsync(CancellationToken cancellationToken = default) =>
             await UpdateStatusAsync(true, cancellationToken).ConfigureAwait(false);
 
+        /// <summary>
+        /// Calls the server to get updated status of the long-running operation.
+        /// </summary>
+        /// <param name="async">When <c>true</c>, the method will be executed asynchronously; otherwise, it will execute synchronously.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>The HTTP response from the service.</returns>
         private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToken cancellationToken)
         {
             if (!_hasCompleted)
             {
-                Response<AnalyzeOperationResult_internal> update = async
-                    ? await _operations.GetAnalyzeReceiptResultAsync(new Guid(Id), cancellationToken).ConfigureAwait(false)
-                    : _operations.GetAnalyzeReceiptResult(new Guid(Id), cancellationToken);
+                var update = async
```

Comment:
could we not use var here? It will be helpful to know what the return type here is

Reply:
No problem! Let me update it.